spec.md

```diff
@@ -1244,13 +1244,14 @@ Default and available values are platform specific. Compose specification MUST s
 
 #### host or none
 
-The syntax for using built-in networks such as `host` and `none` is different, as such networks implicitly exists outside
-the scope of the Compose implementation. To use them one MUST define an external network with the name `host` or `none` and
-an alias that the Compose implementation can use (`hostnet` or `nonet` in the following examples), then grant the service
-access to that network using its alias.
+Use the host’s networking stack, or no networking. Equivalent to `docker run --net=host` or `docker run --net=none`. Only used if you use `docker stack` commands. If you use the `docker-compose` command, use network_mode instead.
```

Comment:
We'd like to make the spec vendor neutral and would prefer to explain things in generic terms rather than just what the current Docker behaviour is.

Reply:
I see! Any suggestions on how to approach this? So idea is to explain these modes without mentioning docker?

Reply:
Networking is going to be a tricky topic as Docker networks do not really map to other platforms. I think this is something that we're going to have to tackle with the community in the fortnightly community meetings.

You can find details about the community meeting on our Slack channel (#compose-specification) on the CNCF Slack (https://slack.cncf.io).

Reply:
In the case of illumos there is the presence of a shared or an exclusive ip stack. You can read the manual section here https://illumos.org/man/5/zones for inspiration. It might help a lot to read how different vendors implement this kind of feature.